config/manager/manager.yaml

```diff
@@ -37,7 +37,7 @@ spec:
               fieldRef:
                 fieldPath: metadata.namespace
           - name: RELATED_IMAGE_OPENJDK
-            value: "quay.io/infinispan/server:13.0.1.Final"
+            value: "quay.io/infinispan/server:13.0.2.Final"
```

Comment:
Shouldn't this be 13.0.3?

Reply:
:+1: It wasn't available when I created the PR and was required for CI. I'll be changing this to 13.0.3.Final as part of the release commit.